server/www/templates/frame/mode-bar.html

```diff
@@ -39,28 +39,163 @@
 
         <ul class="account-menu">
         <!-- user account menu -->
-            <li ng-hide="currentUser" ng-show="canRegister()">
-                <a href="/register" data-modal="signup">
+
+            <li>
+                <a href="http://www.ushahidi.com">
+                    <svg class="ushahidi-logo" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 273.5 266.5" xml:space="preserve">
```

Comment:
Please move this to its own template. Otheriwse this is nearly unreadable.. is it possible to do `<svg src="somefile">` instead?